pkg/network/encoding/format_test.go

```diff
@@ -90,3 +91,40 @@ func BenchmarkConnectionReset(b *testing.B) {
 	}
 	runtime.KeepAlive(c)
 }
+
+func Test_formatProtocols(t *testing.T) {
```

Comment:
maybe add to this UT more complicated test which will show the function aggregated data (as mentioned in the comment of the function and in the example)

Reply:
atm the moment we support a protocol stack of length 1
the protobuf is prepared for the next phases when we will add support for multiple protocols.

at the moment, there is no complicated scenario that I can test